src/com/untamedears/realisticbiomes/RealisticBiomes.java

```diff
@@ -332,15 +342,14 @@ private void registerEvents() {
 			pm.registerEvents(new GrowListener(this), this);
 			pm.registerEvents(new SpawnListener(materialGrowth, fishSpawn), this);
 			pm.registerEvents(new PlayerListener(this, materialGrowth), this);
+			pm.registerEvents(new ClickableInventoryListener(), this);

```

Comment:
I didn't actually see that docu (is there more than the readme?), I saw that when showing a GUI the clicks weren't handled, so I looked into ACivMod and saw that the listener of course has to be registered.